plaso/parsers/mobile_installation.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+"""Parser for iOS mobile installation logs files obtained from iOS device.
+Based on the research from Alex Brignoni."""
```

Comment:
Could you please add a URL referencing the research? I found a GitHub repository but maybe you are referncing something else.

Reply:
Got mixed up in my references.